app/admin/companies.rb

```diff
@@ -84,6 +86,48 @@
       end
 
       tab :cp_assessments do
+        panel 'Carbon Performance Assessments' do
+          if resource.cp_assessments.empty?
+            div class: 'padding-20' do
+              'No Carbon Performance Assessments for this company yet'
```

Comment:
I think we could wrap resources in some kind of presenter object, which could handle things like:
- placeholders for whole resource/empty fields (`No .. yet`, `N/A`, ..)
- formatting dates

What do you think?

Reply:
Good idea. Can try to do that with some SimpleDelegator